### DISCUSSION TITLE
Quarkus Dashboard - problem with displaying jobs

### DISCUSSION THREAD

**Discussion by rafalmurawski** [jobrunr/jobrunr#157] [Q&A]
I downloaded jobrunr/example-quarkus from Your GH repo and conected it to PosgreSQl DB, but when I use Dashboard and i click on JOBS overlap and next group which have at least one job, i see only circle indicator and in IntelliJ console i receive: 2021-06-24 11:37:06,933 SEVERE [org.ecl.yas.int.Unmarshaller] (ForkJoinPool-1-worker-3) Problem adapting object of type class org.jobrunr.jobs.Job to interface javax.json.JsonObject in class class org.jobrunr.utils.mapper.jsonb.JobAdapter

**Comment by rdehuyss** [comment]
Hmmm, that is strange.

Can you setup a github repo that reproduces the issue?

**Reply by rdehuyss** [reply]
I think I know the reason: probably one of your arguments can not be serialized by jsonb. Try to setup a unit test where you serialize and deserialize all of your job parameters.

**Reply by rafalmurawski** [reply]
I don't know why this is happening, but when I ran it on another computer it's ok! Same JDK, same project! Magic!

**Reply by rdehuyss** [reply]
Happy that it is solved!
